### DISCUSSION TITLE
How to use user defined power nets in ORFS flow

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#412] [Q&A]
Hello  OR Team,

In OpenLane flow following option used for power net definition.
```
set ::env(VDD_NETS) [list {vccd1} {vccd2} {vdda1} {vdda2}]
set ::env(GND_NETS) [list {vssd1} {vssd2} {vssa1} {vssa2}]
```

In ORFS flow by default I see from https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/8739eef82b4ec7187291964661c64738f2b89996/flow/platforms/sky130hd/pdn.cfg#L13
```
set ::power_nets "VDD"
set ::ground_nets "VSS"
```
Is there any option to over-write power net definition, so that accordingly PDN created?

**Comment by maliberty** [comment]
You could define PDN_TCL or PDN_CFG but you'll have to replace the whole file.   Alternatively you could modify pdn.cfg to be configurable and make a PR.
